docs/docs/install.rst

```diff
@@ -186,24 +187,34 @@ To create a virtual environment (recommended), run:
     ~> virtualenv .
     ~> source ./bin/activate
 
-Install Isso and its dependencies:
+Install JavaScript modules using ``npm``:
 
 .. code-block:: sh
 
-    ~> python setup.py develop  # or `install`
-    ~> isso run
+    ~> make init
 
-Install JavaScript modules:
+Build JavaScript frontend code:
 
 .. code-block:: sh
 
-    ~> make init
+    ~> make js
 
-Build JavaScript frontend code:
+Install Isso and its dependencies:
 
 .. code-block:: sh
 
-    ~> make js
+    ~> python setup.py develop  # or `pip install -e .`
+    ~> isso run
+
+**Note:** Do not use ``python setup.py install``! That would only install over
```

Comment:
I don't follow this; this does include generated files (including locally changed ones), and I use it regularly.

Reply:
Unfortunate wording on my part.
If isso is installed using `python setup.py install`, the isso artifact files will be copied to `<venv>/lib/python3.9/site-packages/isso-0.12.4-py3.9.egg/isso/`, but any changes you make from now on will not be carried over to the site-packages isso, unless you run the install command again.

Maybe others have better suggestions for the wording? What I want to strongly communicate is that the editable install option will save many from annoying misunderstandings and outdated files.

Reply:
I think this is quite subjective - I don't use venv, and "python install" is how I deploy isso. Perhaps just drop this note altogether? Alternative, just add a caution here about "setup.py install" and venv.

Reply:
I agree that using `pip install -e .` is handy during development since you do not need to install the package again after every change. Since this part of the documentation is about installing from source I think both installation variants are fine and it depends on the use case which one to choose.

So all in all I'd say drop the final `Note: ` line, the rest is just fine.

Reply:
Line dropped